src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -259,6 +259,7 @@ private void Write(ProjectImportedEventArgs e)
         {
             Write(BinaryLogRecordKind.ProjectImported);
             WriteMessageFields(e);
+            Write(e.ImportIgnored);
```

Comment:
Ah, almost forgot. Please increment Microsoft.Build.Logging.BinaryLogger.FileFormatVersion and append to the comment. :)

Reply:
FYI @KirillOsenkov